### ISSUE TITLE
org-indent-mode and version-control-margin don't seem to play well together

### ISSUE THREAD

**Issue by gatl** [syl20bnr/spacemacs#4692]
#### How to reproduce the problem:
1. create a new org file, and place it under version control
2. ensure you have `org-indent-mode` and `version-control-margin` enabled
3. in an indented line, showing a `+` on the right (new line), type a `{` character

#### What I notice:
The line loses its indentation. Further typing makes it restore the indentation, but when closing the bracket the line shifts left again. Further typing may make the line oscillate left and right (e.g., word-ending characters).

#### Work around:
Disabling either `org-indent-mode` or `version-control-margin` makes this problem not manifest itself.

#### System Info
- OS: gnu/linux
- Emacs: 24.5.1
- Spacemacs: 0.105.9
- Spacemacs branch: master (rev. 0562f05)
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
((auto-completion :variables auto-completion-enable-snippets-in-popup t auto-completion-enable-help-tooltip t)
 better-defaults emacs-lisp git markdown org
 (shell :variables shell-default-height 30 shell-default-position 'bottom)
 spell-checking syntax-checking version-control)
```

This seems to be a bizarre problem. Let me know if I was not clear on something.
Thank you.

**Comment by TheBB** [comment]
See https://github.com/syl20bnr/spacemacs/issues/1833

If you switch the marign tool to diff-hl you should not experience this problem.

**Comment by gatl** [comment]
I missed that when searching the issues database. Thank you for pointing that out so quickly.

Where can I find information on how to change (from git-gutter, I assume) to diff-hl?

**Comment by TheBB** [comment]
See the version-control layer README.

**Comment by syl20bnr** [comment]
We disabled org indent by default, I consider this "fixed".

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
